fortran/src/H5ESff.F90

```diff
@@ -0,0 +1,296 @@
+!> @defgroup FH5ES Fortran Event Set (H5ES) Interface
+!!
+!! @see H5ES, C-API
+!!
+!! @see @ref H5ES_UG, User Guide
+!!
+!
+! COPYRIGHT
+! * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+!   Copyright by The HDF Group.                                               *
+!   All rights reserved.                                                      *
+!                                                                             *
+!   This file is part of HDF5.  The full HDF5 copyright notice, including     *
+!   terms governing use, modification, and redistribution, is contained in    *
+!   the COPYING file, which can be found at the root of the source code       *
+!   distribution tree, or in https://www.hdfgroup.org/licenses.               *
+!   If you do not have access to either file, you may request a copy from     *
+!   help@hdfgroup.org.                                                        *
+! * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+!
+
+MODULE H5ES
+
+  USE, INTRINSIC :: ISO_C_BINDING, ONLY : C_PTR, C_CHAR, C_INT64_T, C_BOOL
+  USE H5GLOBAL
+  IMPLICIT NONE
+
+CONTAINS
+
+!>
+!! \ingroup FH5ES
+!!
+!! \brief Creates an event set.
+!!
+!! \param es_id  \es_id
+!! \param hdferr \fortran_error
+!!
+!! See C API: @ref H5EScreate()
+!!
+  SUBROUTINE H5EScreate_f(es_id, hdferr)
```

Comment:
Our naming convention was to use low case for the names of the Fortran subroutines, e.g., h5fcreate_f. Should we stick to it?

Reply:
Sure, I was lazy and just copied/paste from the reference manual.